extras/dgoss/dgoss

```diff
@@ -66,6 +68,8 @@ run(){
     esac
 
     $CONTAINER_RUNTIME logs -f "$id" > "$tmp_dir/docker_output.log" 2>&1 &
+    # There is a chance that the log will not be written, an empty line is added for mitigation.
+    echo "" >> "$tmp_dir/docker_output.log" 2>&1 &
```

Comment:
Perhaps `touch` makes more sense here?

Reply:
I will test and check this.

Reply:
I have looked at it. `touch` does not help. I think I have found an better and more transparent solution.